Pollo/ViewControllers/CardController.swift

```diff
@@ -170,6 +170,7 @@ class CardController: UIViewController {
         
         if userRole == .admin {
             createPollButton = UIButton()
+            createPollButton.isHidden = pollsDateModel.polls.last?.state == .live
```

Comment:
should this be `= session.isLive` ?

Reply:
For consistency, we could use it but within `cardController`, `pollsDateModel` holds the same data and is always correct. The fact that we are altering `session.isLive` made me only want to use it when it was a necessity.